README.md

```diff
@@ -69,7 +69,9 @@ Installing the app will yield you with a webhook that the app uses to publish up
 
 ### Environment variables
 
-The `SLACK_WEBHOOK` variable has to be defined in the environment where you will be running semantic release. This can be done by exporting it in bash or in the user interface of you CI provider. Obtain this token by installing the slack app according to [slack app installation](#slack-app-installation).
+The `SLACK_WEBHOOK` variable can be defined in the environment where you will be running semantic release. This can be done by exporting it in bash or in the user interface of you CI provider. Obtain this token by installing the slack app according to [slack app installation](#slack-app-installation).
```

Comment:
Just a spelling mistake from before `...of you CI...` should probably be `...of your CI...`

Reply:
Yup, will change it